content/en/docs/distributions/gke/deploy/upgrade.md

```diff
@@ -140,24 +213,40 @@ General instructions for upgrading Kubeflow:
       kubectl config use-context ${KF_NAME}
       ```
 
-1. Edit the Makefile at `./Makefile` and change `MANIFESTS_URL` to point at the version of Kubeflow manifests you want to use
+1. Merge the new version of `kubeflow/gcp-blueprints` (example: v1.3.0), you don't need to do it again if you have already done so during management cluster upgrade.
+
+   ```bash
+   WORKING_BRANCH=<your-github-working-branch>
+   VERSION_TAG=<targeted-kubeflow-version-tag-on-github>
+   git checkout "${WORKING_BRANCH}"
+   git merge "${VERSION_TAG}"
```

Comment:
I think we miss a step to fetch remote, `git fetch upstream` or `git fetch origin`

Reply:
Make sense! Updated.